### PR TITLE
Processing interface

### PR DESCRIPTION
**Pull request by josenavas**
This PR adds the processing interface, already connected with the backend.
Only 2 missing points:
 - [x] Remove jobs from the workflow
 - [x] Add JavaScript documentation

### CODE REVIEW

File: qiita_pet/handlers/api_proxy/processing.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    artifact = Artifact(artifact_id)
+
+    return {'status': 'success',
+            'message': '',
+            'name': artifact.name,
+            'type': artifact.artifact_type}
+
+
+def list_commands_handler_get_req(artifact_types):
+    """Retrieves the commands that can process the given artifat types
```
Comment (antgonza) [inline qiita_pet/handlers/api_proxy/processing.py]: artifact

File: qiita_pet/handlers/api_proxy/processing.py [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    -------
+    dict of objects
+        A dictionary containing the commands information
+        {'status': str,
+         'message': str,
+         'commands': list of dicts of {'id': int,
+                                       'command': str,
+                                       'output': list of [str, str]}}
+    """
+    artifact_types = artifact_types.split(',')
```
Comment (antgonza) [inline qiita_pet/handlers/api_proxy/processing.py]: Just making sure this is safe ==> artifact types can't have commas.

File: qiita_pet/handlers/api_proxy/processing.py (hunk 1 above)
Comment (josenavas) [inline qiita_pet/handlers/api_proxy/processing.py]: Done

File: qiita_pet/handlers/api_proxy/processing.py (hunk 2 above)
Comment (josenavas) [inline qiita_pet/handlers/api_proxy/processing.py]: Correct

**Comment by josenavas** [comment]
Ready for review/merge!

File: qiita_pet/templates/study_ajax/processing_artifact.html [hunk 3]
```diff
@@ -0,0 +1,477 @@
...
+          });
+      });
+  }
+
+  /*
+   * Generates the GUI for selecting the commands to apply to the given artifacts
+   *
+   * @param p_nodes list The ids of the selected artifacts
+   *
+   * This function executes and AJAX call to retrieve all the commands that can
```
Comment (antgonza) [inline qiita_pet/templates/study_ajax/processing_artifact.html]: and -> an

**Comment by antgonza** [comment]
:+1: @mortonjt @ElDeveloper could you take a look?

**Comment by ElDeveloper** [comment]
:+1:, ready for merge once tests pass.

**Comment by josenavas** [comment]
@antgonza @ElDeveloper wanna merge?

**Pull request merged**